doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -12,13 +12,15 @@ The ``Scheme`` command is a high-level tool for generating automatically
 (possibly mutual) induction principles for given types and sorts. Its
 syntax follows the schema:
 
-.. cmd:: Scheme @ident := Induction for @ident Sort sort {* with @ident := Induction for @ident Sort sort}
+.. cmd:: Scheme @ident := Induction for @ident' Sort sort {* with @ident := Induction for @ident Sort sort}
```

Comment:
`@ident` supports subscript `__1` `__2` ... (notice the two `_`).
You could write for instance `Scheme @ident__1 := Induction for @ident__2 Sort sort {* with @ident__i := Induction for @ident__j Sort sort}`